### DISCUSSION TITLE
Feature to go back and retry upon completion or failure

### DISCUSSION THREAD

**Discussion by SvCy** [lncrawl/lightnovel-crawler#2426] [Ideas]
I think it'll be great to able to go back to the novel/sites list to retry from a different source upon completion or failure for we often retry from another site as the downloaded file may have chapter issues due to it's source website.
e.g.
>Open folder
>Go back to the list
>Exit

Otherwise, we need to search again for another 5 mins or so just for the list. and then again if luck is bad. (can stop extra searches as well). So, what do you think?
Thank You for reading..

**Comment by dipu-bd** [comment]
Ideally, you should search on the internet and grab the link from there. Or, you can save the links displayed after a search, and use it later.

**Reply by SvCy** [reply]
I do, but not all sites are supported. Eg Ranobes, Panda, ... That's why searching is/should've been an ideal way.
That feature could be very convenient, but ig i got your answer.
& Thanks a lot for the tool!!🔥
